### ISSUE TITLE
No tweener module starting from GNOME 3.37

### ISSUE THREAD

**Issue by docquantum** [mlutfy/hidetopbar#215]
While I know 3.37 is the dev version of gnome, this indicates that upgrading to 3.38 may mean some refactoring is in order. I'm no expert in shell extension (although, I'd like to learn), but here is the relevant failure message from journalctl upon trying to load the extension:

```
JS ERROR: Extension hidetopbar@mathieu.bidon.ca: ImportError: No JS module 'tweener' found in search path
@/home/user/.local/share/gnome-shell/extensions/hidetopbar@mathieu.bidon.ca/panelVisibilityManager.js:10:17
@/home/user/.local/share/gnome-shell/extensions/hidetopbar@mathieu.bidon.ca/extension.js:13:32
_callExtensionInit@resource:///org/gnome/shell/ui/extensionSystem.js:397:13
loadExtension@resource:///org/gnome/shell/ui/extensionSystem.js:331:27
activate@/home/dshchur/.local/share/gnome-shell/extensions/extension-reloader@nls1729/extension.js:112:34
vfunc_button_release_event@resource:///org/gnome/shell/ui/popupMenu.js:138:14

```

If any more information is needed, I'll be happy to provide.

**Comment by docquantum** [comment]
It looks like tweener was dropped in favor of Clutter's own animation system back in 3.34, however, they only now removed it with https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/1200

**Comment by tuxor1337** [comment]
Thanks for early notice!

**Comment by tuxor1337** [comment]
Fixed in aea14bf
